src/middlewares/index.js

```diff
@@ -0,0 +1,3 @@
+
+// eslint-disable-next-line
```

Comment:
Why is this here? I thought we have an `eslint` configuration file where our rules are declared